### PR TITLE
Fix Ecto.Changeset.check_constraint/3 @doc example

### PR DESCRIPTION
**Pull request by esambo**
I just noticed that the example given from #2878 doesn't work and fixed it accordingly

### CODE REVIEW

File: lib/ecto/changeset.ex [hunk 1]
```diff
@@ -2307,14 +2307,14 @@ defmodule Ecto.Changeset do
   In order to use the check constraint, the first step is
   to define the check constraint in a migration:
 
-      create constraint("users", :price_must_be_positive, check: "price > 0")
+      create constraint("users", :price, :price_must_be_positive, check: "price > 0")
```
Comment (wojtekmach) [inline lib/ecto/changeset.ex]: this was correct for _migrations_, could you revert this change? Good catch on the changeset function tho!

Comment (esambo) [reply lib/ecto/changeset.ex]: ups, thanks for catching that! I fixed it in 5df8138

**Comment by wojtekmach** [comment]
Thank you!
